config/settings/base.py

```diff
@@ -139,9 +136,18 @@
 LOGIN_URL = '/askup/sign-in'
 
 EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
-EMAIL_HOST = 'postfix'
+DEFAULT_FROM_EMAIL = 'AskUp mailer <mailer@askup.net>'
+
+# Default secure settings
+EMAIL_HOST = 'localhost'
 EMAIL_PORT = 25
 EMAIL_HOST_USER = ''
 EMAIL_HOST_PASSWORD = ''
 EMAIL_USE_TLS = False
-DEFAULT_FROM_EMAIL = 'AskUp mailer <mailer@askup.net>'
+
+SECRET_KEY = '(+wig4^c1749^^(23rp3$8y8r8_9o%-=#ujjfkasbjl=)-#b&o'
```

Comment:
Please use default values from the secure_example.py

Reply:
done